helmfile.d/values/ingress-nginx.yaml.gotmpl

```
@@ -94,6 +97,9 @@ controller:
   resources: {{- toYaml .Values.ingressNginx.controller.resources | nindent 4 }}
 
   service:
+    {{ if not .Values.ingressNginx.controller.service.allocateLoadBalancerNodePorts }}
+    allocateLoadBalancerNodePorts: false
+    {{ end }}
```

```suggestion
    allocateLoadBalancerNodePorts: {{ .Values.ingressNginx.controller.service.allocateLoadBalancerNodePorts }}
```
Since it is part of the default config anyway.

---

I guess since it is default true in Kubernetes, but if you don't set it at all it will not show up in helm diff with how it is configured in the [upstream chart](https://github.com/kubernetes/ingress-nginx/blob/main/charts/ingress-nginx/values.yaml#L475), which it will do if we change the templating:
```
  spec:
    type: LoadBalancer
+   allocateLoadBalancerNodePorts: true
```
But I guess that is a not really a problem :sweat_smile: